packages/wallets/src/solana/wallet.ts

```diff
@@ -85,11 +89,12 @@ export class SolanaSmartWallet extends SolanaWallet {
         apiClient: ApiClient,
         publicKey: PublicKey,
         adminSignerInput: SolanaSignerInput,
+        callback: Callbacks = {},
```

Comment:
should it be called `callbacks` in plural?

Reply:
nice catch https://github.com/Crossmint/crossmint-sdk/pull/985/commits/fb2d5bd15ba95fdb21757a7245dec828c474c2e8